examples/rvc-app/rvc-common/src/rvc-device.cpp

```diff
@@ -160,6 +224,35 @@ void RvcDevice::HandleOpStateGoHomeCallback(Clusters::OperationalState::GenericO
     }
 }
 
+bool RvcDevice::HandleIsSetSelectedLocationCallback(std::string & locationText)
```

Comment:
first path template - feel free to update for whatever desired example behavior is.